<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>373</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>373</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Asignar materias al alumno</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>241</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nombre  de la cuenta</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="NombreCuenta"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nombre de la Materia</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QComboBox" name="AbrMat"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Grupo</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QComboBox" name="Grupo"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Semestre</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QComboBox" name="Semestre"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Fecha</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="Fecha"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>205</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Insertar">
        <property name="text">
         <string>Insertar Materia</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Ver">
        <property name="text">
         <string>Ver Tabla</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="VerCuentas">
      <property name="text">
       <string>Ver Tabla de cuentas de usuario</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
